bin/spark-shell

```diff
@@ -35,7 +35,8 @@ export _SPARK_CMD_USAGE="Usage: ./bin/spark-shell [options]"
 # so we need to add the "-Dscala.usejavacp=true" flag manually. We
 # do this specifically for the Spark shell because the scala REPL
 # has its own class loader, and any additional classpath specified
-# through spark.driver.extraClassPath is not automatically propagated.
+# through spark.driver.extraClassPath and/or spark.common.extraClassPath are not
```

Comment:
(Nevermind, the option is to enable those spark options to be respected by the shell.)